test/unit/action/info.spec.js

```diff
@@ -16,11 +18,12 @@ describe('Action Info Unit Tests', () => {
     sandbox = sinon.createSandbox({});
     sandbox.stub(logger);
     store = new Store();
+    nav = sinon.createStubInstance(NavAction);
     require('../../../src/config').RETRY_DELAY = 1;
     grpc = sinon.createStubInstance(GrpcAction);
     grpc.sendCommand.resolves({});
     notification = sinon.createStubInstance(NotificationAction);
-    info = new InfoAction(store, grpc, notification);
+    info = new InfoAction(store, grpc, nav, notification);
```

Comment:
You need to also add this argument in the `InfoAction` constructor in the integration tests.